src/flask_cognito_lib/decorators.py

```diff
@@ -252,6 +270,13 @@ def wrapper(*args, **kwargs):
                     domain=cognito_auth.cfg.cookie_domain,
                 )
 
+            # Remove the id token if it exists
+            if get_token_from_cookie(cognito_auth.cfg.COOKIE_NAME_ID):
+                resp.delete_cookie(
+                    key=cognito_auth.cfg.COOKIE_NAME_ID,
```

Comment:
Would you be able to add some test coverage of these new conditional statements please?

Reply:
See commit below